### DISCUSSION TITLE
Is the json serialization of this library interchangable?

### DISCUSSION THREAD

**Discussion by kkpgarcia** [atteneder/glTFast#425] [Q&A]
I ran into a problem where exported models that includes the character `"` breaks the json structure of gltf json. I'm wondering if there are ways to extend or change how the serialization behaves in this library.

**Comment by atteneder** [comment]
JSON serialization is implemented in the `GLTFast.Schema` namespace classes (search for `public void GltfSerialize(StreamWriter stream)`). It does not use a framework or library whatsoever.

I assume you have objects with a `"` in the name and those are not properly escaped. I suggest to open an issue and include a small test scene.

Thanks

**Comment by kkpgarcia** [comment]
Is there a way to extend or override the serialization process though?

**Reply by atteneder** [reply]
Not yet, but that'll probably get added at some point in order to support custom extensions.
If you're willing to explain your use-case, I can consider it and might raise the priority on it.
